<launch>


  <node pkg="multi_robot" name="home" type="home.py"/>

  <!--node pkg="multi_robot" name="move_close" type="move_close.py"/-->

  <node pkg="multi_robot" name="send_mani" type="send_mani.py"/>
  <node pkg="multi_robot" name="aruco_move" type="aruco_move.py"/>
  <node pkg="multi_robot" name="self_drive" type="self_drive.py"/>



</launch>
